crates/ursa-application/src/types.rs

```diff
@@ -9,9 +9,8 @@ use abci::{
     types::*,
```

Comment:
It would be useful to review the unwraps used in this crate, and maybe use Result instead, since one of these methods turned out to be fallible.

Reply:
anyhow::Result would be a great addition

Reply:
Cleaned up the init_chain function a bit. If we cant load genesis we should panic and loading in static bytes should not ever, but cant return anyhow::Result in this particular function because its part of the abci spec. Genesis refactor on the roadmap the plan is to move loading genesis out of this function and load it directly into parity db on the app creation, then we can switch it up to anyhow.